py/embedded-server/deephaven_server/cli.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+#
+import click
+import signal
+import sys
+import webbrowser
+
+from .server import Server
+
+@click.group()
+def cli():
+    """
+    Command-line interface entry point for the Deephaven embedded server application.
+    Accepts a command to start the application and the arguments to use.
+    """
+    pass
+
+
+@cli.command()
+@click.option("--host", default=None, help="The host to bind to.")
+@click.option("--port", default=None, type=int, help="The port to bind to.")
+@click.option("--jvm-args", default=None, help="The JVM arguments to use.")
+def server(host, port, jvm_args):
+    """
+    Start the Deephaven server.
+    """
+    click.echo("Starting Deephaven server...")
+
+    if jvm_args is None:
+        jvm_args = ""
+    jvm_args = jvm_args.split()
+
+    s = Server(host=host, port=port, jvm_args=jvm_args)
+    s.start()
```

Comment:
Is there any good reason to not support the `extra_classpath` argument recently added to `Server()`?

Reply:
That was added while this PR was in flight. I don't see why not, and I think @devinrsmith had that intention.

Reply:
@jmao-denver added